src/rpc/handlers/ServerInfo.cpp

```diff
@@ -48,7 +48,7 @@ doServerInfo(Context const& context)
     if (admin)
     {
         info[JS(counters)] = boost::json::object{};
-        info[JS(counters)].as_object()[JS(rpc)] = context.counters.report();
```

Comment:
Is it intended that it seems to be stored in a different path?

Reply:
I just moved the addition of the "rpc" object inside of `Counters::report()`, so it should ultimately still be in the same path.